src/main/java/frc/robot/lib/Swerve.java

```diff
@@ -23,7 +24,7 @@
 /** Add your docs here. */
 public class Swerve {
 
-    public static SwerveDriveOdometry swerveOdometry;
+    public static SwerveDrivePoseEstimator swerveOdometry;
```

Comment:
As a part of this upgrade, you should also migrate the logic in PhotonCameraWrapper.js into this class, since all that class does is update the pose of a SwerveDrivePoseEstimator object.